recipes/lsst-ts-xml/meta.yaml

```
@@ -0,0 +1,54 @@
+{% set name = "lsst-ts-xml" %}
+{% set version = "20.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/lsst-ts/ts_xml/archive/v{{ version }}.tar.gz
+  sha256: 04a8c8053c1c26180b689c61ab470de8117f6ac243158f17b17aa1a288dfc5fe
+
+build:
+  skip: True  # [win, py<310]
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools_scm
+  run:
+    - python
+    - astropy
```

```suggestion
    - astropy
    - _unix
```
This will ensure it only installs on osx and Linux.

---

It has to be with a double underscore.